src/couch.js

```diff
@@ -12,23 +15,66 @@ async function _request(token, path, options) {
   return await response.json();
 }
 
+async function _requestAll(token, path, content) {
```

Comment:
There might be a way to incorporate POST requests and request payloads into the `_request` method, instead of creating a new method.

Reply:
I am not sure about it yet,  I am yet to find a way to figure that out, But i have a new commit in the PR which resolves other comments, and gets the List of found and Not Found ID's

Reply:
I think it might be productive for us to have a call about this tomorrow and go through how we'd refactor this code.